From 4df5bbdc7c54f602b0ba4e31f97aa64e1cf3395e Mon Sep 17 00:00:00 2001
From: James Bonfield <jkb@sanger.ac.uk>
Date: Tue, 16 Mar 2021 15:26:21 +0000
Subject: [PATCH] Fixed --enable-custom-buffering for cram_bambam.

This configure option adjusts io_lib/os.h, so the C code needs to
include it for checking.  Reported with fix by German Tischler.

Fixes #39
---
 htscodecs            | 2 +-
 io_lib/cram_bambam.c | 1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/htscodecs b/htscodecs
index 6dadc2a..ae9cfb7 160000
--- a/htscodecs
+++ b/htscodecs
@@ -1 +1 @@
-Subproject commit 6dadc2a8d42d4d101b0bc96807147fce96fe0ddf
+Subproject commit ae9cfb7dcc3f5e1c07ce7f21b4fec4b3f7c49c60
diff --git a/io_lib/cram_bambam.c b/io_lib/cram_bambam.c
index 7a7298c..2e85fbf 100644
--- a/io_lib/cram_bambam.c
+++ b/io_lib/cram_bambam.c
@@ -64,6 +64,7 @@
 // not used
 #else
 
+#include "io_lib/os.h"
 #include "io_lib/cram_bambam.h"
 
 //-----------------------------------------------------------------------------
-- 
2.31.0

